### ISSUE TITLE
Figure out how to make package.json scripts work on Windows

### ISSUE THREAD

**Issue by bebraw** [survivejs/webpack_react#8]
Ideally there should be some cross-platform way to achieve this.

Related to https://github.com/survivejs/webpack_react/commit/f2f4b35c081c3b6c5c3cd0b9cf32e138311dce3d#commitcomment-11029492 .

**Comment by bebraw** [comment]
Unfortunately there's no simple cross-platform way to do this as far as I can see. I added an alternative solution and a warning about this at the book. Hopefully Webpack 2 will fix this (allows params to be passed directly).

**Comment by bebraw** [comment]
@JaxCavalera I worked around this through npm. `const TARGET = process.env.npm_lifecycle_event;
` gives us the information we need. I.e. if you run **npm run build**, `TARGET` will point at `build`. You can do branching etc. based on that.

So the solution was simply to avoid using env variables. Simple as that.

**Comment by JaxCavalera** [comment]
@bebraw yep that turned out to be the general gist of how I resolved it also.  bit different i how i made it work but the underlying approach of avoiding env vars for the task is the same.  I just have pointed to the wrong target or something I think but I've got that all proposed on reactiflux so hopefully can get it resolved.. and i'll stop here since this is no longer related to using environment variables for the task of multiple webpack configurations.
